torch/distributed/rendezvous.py

```diff
@@ -41,8 +41,22 @@ def register_rendezvous_handler(scheme, handler):
     _rendezvous_handlers[scheme] = handler
 
 
-def rendezvous(url, **kwargs):
-    global _rendezvous_handlers
+def rendezvous(url, rank=-1, world_size=-1, **kwargs):
```

Comment:
This is changing the `rendezvous` Python API. @pietern do you know if anyone using this API directly?